src/components/colorways/index.js

```diff
@@ -1,54 +1,13 @@
 import dsa_galaxy_class from './dsa/galaxy_class';
-import sa_carbon from './sa/carbon';
-import sa_danger_zone from './sa/danger_zone';
-import sa_jukebox from './sa/jukebox';
-import sa_modern_selectric from './sa/modern_selectric';
-import sa_nantucket_selectric from './sa/nantucket_selectric';
-import sa_oblivion_hagoromo from './sa/oblivion_hagoromo';
-import gmk_dolch from './gmk/dolch';
-import gmk_jamon from './gmk/jamon';
-import gmk_merlin from './gmk/merlin';
-import gmk_metaverse from './gmk/metaverse';
-import gmk_mizu from './gmk/mizu';
-import gmk_nautilus from './gmk/nautilus';
-import gmk_olivetti from './gmk/olivetti';
-import gmk_olivia from './gmk/olivia';
-import gmk_serika from './gmk/serika';
-import gmk_space_cadet from './gmk/space_cadet';
-import gmk_striker from './gmk/striker';
-import gmk_ta_royal_alpha from './gmk/ta_royal_alpha';
-import gmk_wob from './gmk/wob';
-import sa_vilebloom from './sa/vilebloom';
+import gmk from './gmk';
+import sa from './sa';
 import alphaCodes from './alphaCodes';
 import modCodes from './modCodes';
 
 export default {
   alphaCodes,
   modCodes,
-  list: [
-    dsa_galaxy_class,
-    sa_carbon,
-    sa_danger_zone,
-    sa_jukebox,
-    sa_modern_selectric,
-    sa_nantucket_selectric,
-    sa_oblivion_hagoromo,
-    sa_vilebloom,
-
-    gmk_dolch,
-    gmk_jamon,
-    gmk_merlin,
-    gmk_metaverse,
-    gmk_mizu,
-    gmk_nautilus,
-    gmk_olivetti,
-    gmk_olivia,
-    gmk_serika,
-    gmk_space_cadet,
-    gmk_striker,
-    gmk_ta_royal_alpha,
-    gmk_wob
-  ],
+  list: [dsa_galaxy_class, ...gmk, ...sa],
```

Comment:
```suggestion
  list: [dsa_galaxy_class, ...sa, ...gmk],
```

Makes it so the ordering stays the same in the UI.

Reply:
oh, fair enough. I was going for alpha